org-node/src/client.rs

```diff
@@ -587,6 +587,21 @@ impl Client {
                     )
                     .map_err(|e| Error::SetHead(Box::new(e)))?;
 
+                let reference = repository
+                    .find_reference(local_branch_ref.to_str().unwrap_or_default())
+                    .map_err(|e| Error::SetHead(Box::new(e)))?;
+
+                let oid = reference.target().expect("reference target must exist");
+
+                // Announce the updated refs to the network.
+                if let Err(e) = api.announce(Payload {
+                    urn,
+                    rev: Some(Rev::Git(oid)),
+                    origin: Some(api.peer_id()),
+                }) {
+                    tracing::error!(target: "org-node", "Error announcing refs: {:?}", e);
```

Comment:
```suggestion
                    tracing::error!(target: "org-node", "Error announcing refs: {}", e);
```

Reply:
Error is payload type, so it does not implement display, maybe can use `e.urn.encode_id()` ?

Reply:
Oh weird, let's leave it like this for now.